### PR TITLE
feat: Release/v1.15.0

### PR DESCRIPTION
**Pull request by vsukhin**
## Checklist (choose whats happened)

- [ ] breaking change! (describe)
- [ ] tested locally
- [ ] tested on cluster
- [ ] added new dependencies
- [ ] updated the docs
- [ ] added a test

## Breaking changes

-

## Changes

-

## Fixes

-

### CODE REVIEW

File: .github/workflows/docker-build-develop.yaml [hunk 1]
```diff
@@ -22,15 +22,18 @@ jobs:
         id: buildx
         uses: docker/setup-buildx-action@v1
 
-      - name: Go Cache
-        uses: actions/cache@v2
+      # - name: Go Cache
```
Comment (nicufk) [inline .github/workflows/docker-build-develop.yaml:25]: don't we need cache anymore?

File: .github/workflows/docker-build-release.yaml [hunk 2]
```diff
@@ -23,15 +23,15 @@ jobs:
...
-      - name: Go Cache
-        uses: actions/cache@v2
-        with:
-          path: |
-            ~/go/pkg/mod
-            ~/.cache/go-build
-          key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
-          restore-keys: |
-            ${{ runner.os }}-go-
+#      - name: Go Cache
```
Comment (nicufk) [inline .github/workflows/docker-build-release.yaml:26]: ?

File: Makefile [hunk 3]
```diff
@@ -80,12 +89,6 @@ docker-build-api:
 docker-build-cli:
 	env SLACK_BOT_CLIENT_ID=** SLACK_BOT_CLIENT_SECRET=** ANALYTICS_TRACKING_ID=** ANALYTICS_API_KEY=** SEGMENTIO_KEY=** CLOUD_SEGMENTIO_KEY=** DOCKER_BUILDX_CACHE_FROM=type=registry,ref=docker.io/kubeshop/testkube-cli:latest ALPINE_IMAGE=alpine:3.18.0 goreleaser release -f .builds-linux.goreleaser.yml --rm-dist --snapshot
 
-
```
Comment (nicufk) [inline Makefile:83]: how do we build it locally now?

File: cmd/kubectl-testkube/commands/cloud/init.go [hunk 4]
```diff
@@ -18,7 +18,7 @@ func NewInitCmd() *cobra.Command {
 
 	cmd := &cobra.Command{
 		Use:     "init",
-		Short:   "Install Helm chart registry in current kubectl context and update dependencies",
+		Short:   "Install Testkube Cloud Agent and connect to Testkube Cloud environment",
```
Comment (nicufk) [inline cmd/kubectl-testkube/commands/cloud/init.go:21]: Do we want to have same message for the oss?

**Review by nicufk — APPROVED** [review]
it was long 😄, looks good, some comments

File: .github/workflows/docker-build-develop.yaml (hunk 1 above)
Comment (vsukhin) [reply .github/workflows/docker-build-develop.yaml]: we have some caching issues using latest go version in gorealeser

File: .github/workflows/docker-build-release.yaml (hunk 2 above)
Comment (vsukhin) [reply .github/workflows/docker-build-release.yaml:26]: we will return it back after while

File: Makefile (hunk 3 above)
Comment (vsukhin) [reply Makefile:83]: good question for @ypoplavs

**Pull request merged**